app/cli/cmd/diffs.go

```
@@ -1,28 +1,48 @@
 
 func init() {
 	RootCmd.AddCommand(diffsCmd)
 
 	diffsCmd.Flags().BoolVarP(&plainTextOutput, "plain", "p", false, "Output diffs in plain text with no ANSI codes")
+	diffsCmd.Flags().BoolVar(&showDiffUi, "ui", false, "Show diffs in a browser UI")
+	diffsCmd.Flags().BoolVar(&diffGit, "git", true, "Show diffs in git diff format")
+	diffsCmd.Flags().BoolVarP(&diffUiSideBySide, "side", "s", true, "Show diffs UI in side-by-side view")
+	diffsCmd.Flags().BoolVarP(&diffUiLineByLine, "line", "l", false, "Show diffs UI in line-by-line view")
+
+	diffsCmd.Flags().BoolVar(&fromTellMenu, "from-tell-menu", false, "Show diffs from the tell menu")
+	diffsCmd.Flags().MarkHidden("from-tell-menu")
```

golangci-lint (1.64.8)

Error return value of `(*github.com/spf13/pflag.FlagSet).MarkHidden` is not checked

(errcheck)